android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -201,47 +185,36 @@ private void setupFragmentManager() {
   }
 
   protected FragmentTransaction getOrCreateTransaction() {
-    if (mCurrentTransaction == null) {
-      mCurrentTransaction = mFragmentManager.beginTransaction();
-      mCurrentTransaction.setReorderingAllowed(true);
+    if (mCurrentTransaction != null) {
```

Comment:
this seems to no longer be necessary if we create new transaction and then commit it with each single operation. As a consequence we wouldn't need the `mCurrentTransaction` field nor need to call getOrCreaeTransaction nor tryCommitTransaction

Reply:
I think it was left for usage with `ScreenStack`, where we have animations in `Fragments`, so we want to concatenate all the operations into one and run animations on them. Btw, I would change the name `commitNowAllowingStateLoss` to something like `commitTransaction`.